ui/src/features/stage/stage-details.tsx

```diff
@@ -123,6 +124,7 @@ export const StageDetails = ({ stage }: { stage: Stage }) => {
       return [];
     }
   }, [argocdShard, stage]);
+  const { stageTabs } = useExtensionsContext();
```

Comment:
@rpelczar @Marvin9 the `argocdLinks` stuff above this line has moved out of this file in `main`, so I'm not 100% sure that this new line is in the right place.